test/evergreen.yml

```diff
@@ -413,51 +413,51 @@ functions:
   "many dbs test":
     command: shell.exec
     params:
-      working_dir: "wiredtiger/build_posix/test/manydbs"
+      working_dir: "wiredtiger/cmake_build/test/manydbs"
       script: |
         set -o errexit
         set -o verbose
-        ${test_env_vars|} ./t ${many_db_args|} 2>&1
```

Comment:
Have we been running `t` instead of `test_manydbs`?

Reply:
Autoconf produces a binary named `t`, whilst CMake produces a binary named `test_manydbs`. This change updates to the expected binary name in CMake.